client/extensions/woocommerce/state/sites/settings/email/selectors.js

```diff
@@ -147,6 +153,27 @@ export const isRequestingSyncStatus = ( state, siteId ) => {
 };
 
 /**
+ * Returns true if MailChimp plugin is configured correctly and has connection to server
```

Comment:
to which server? MailChimp's? please reword as "...and has connection to MailChimp servers"